README.md

```diff
@@ -11,7 +11,7 @@ urlFragment: storage-blobs-node-quickstart
 # Storage Blobs Quickstart
 
 > **WARNING** 
-> You are viewing a tutorial for the legacy version of the Azure Storage Node.js SDK. Please go [here](https://github.com/Azure-Samples/azure-storage-js-v10-quickstart) to view the latest JavaScript SDK (v10).
+> You are viewing a tutorial for the legacy version of the Azure Storage Node.js SDK. Please go [here](https://github.com/Azure-Samples/azure-storage-js-v10-quickstart) to view the latest JavaScript SDK (v10 and v12).

```

Comment:
We now have a standard template for adding such messages in the sample repo. Please use this format in future PRs as well.

```suggestion
> Samples in this repository use older package `azure-storage` for Azure Storage Blobs.
We recommend that you refer to the [samples that use the new package](https://github.com/Azure-Samples/azure-sdk-for-js-storage-blob-upload-download) `@azure/storage-blob` instead.
```

Reply:
Ok, Readme is updated, please have a review.